applications/plugins/image/ImageSampler.h

```diff
@@ -80,7 +80,7 @@ struct ImageSamplerSpecialization<defaulttype::Image<T>>
 
     static void regularSampling( ImageSamplerT* sampler, const bool atcorners=false, const bool recursive=false )
     {
-//        typedef typename ImageSamplerT::Real Real;
+        //        typedef typename ImageSamplerT::Real Real;
```

Comment:
remove

Reply:
The pr is about sout/serr, not removing dead code (even if I did it)